BSP file - device-tree

This patch exposes the CR10 led gpio present on the Parallella board to the sysfs.

Signed-off-by: Theodor Gherzan <theodor@resin.io>

--- common/zynq7-base.dtsi	2015-04-08 13:43:16.005351615 +0200
+++ common/zynq7-base.dtsi	2015-04-08 13:46:06.381411359 +0200
@@ -498,4 +498,12 @@
 			interrupts = <0 7 4>;
 		} ;
 	} ;
+	leds {
+		compatible = "gpio-leds";
+
+		CR10 {
+			gpios = <&ps7_gpio_0 7 0>;
+			default-state = "off";
+		} ;
+	} ;
 } ;
